storage/badger/qcs_test.go

```diff
@@ -0,0 +1,58 @@
+package badger_test
```

Comment:
could we add a test for verifying that the expected `ErrNotFound` is returned if no QC for the given block is known? thanks